src/gwt/src/org/rstudio/studio/client/workbench/addins/AddinsCommandManager.java

```diff
@@ -104,26 +111,46 @@ public void execute(EditorKeyBindings bindings)
    private void registerBindings(final EditorKeyBindings bindings,
                                  final CommandWithArg<EditorKeyBindings> afterLoad)
    {
-      List<Pair<List<KeySequence>, CommandBinding>> commands =
-            new ArrayList<Pair<List<KeySequence>, CommandBinding>>();
-      
-      for (String id : bindings.iterableKeys())
-      {
-         List<KeySequence> keyList = bindings.get(id).getKeyBindings();
-         CommandBinding binding = new AddinCommandBinding(id);
-         commands.add(new Pair<List<KeySequence>, CommandBinding>(keyList, binding));
-      }
-      
-      KeyMap map = ShortcutManager.INSTANCE.getKeyMap(KeyMapType.ADDIN);
-      for (int i = 0; i < commands.size(); i++)
+      server_.getRAddins(false, new ServerRequestCallback<RAddins>()
```

Comment:
This ensures that saved keybindings for addins enter the keymap when RStudio is loaded. Should I be using `true` here to ensure that the callback gets the up-to-date definition of addins?